services/portal/studyViewer/studyViewerProps.js

```diff
@@ -1,17 +1,26 @@
 module.exports = {
   path: '/study-viewer/clinical_trials',
   datasetPath: '/study-viewer/clinical_trials/ACTT',
+  datasetPath1: 'study-viewer/clinical_trials/NCT04401579',
   studyViewerDivClass: '.study-viewer',
+  // TODO : improve the selector
+  studyViewerRelPath: '//body/div[@id=\'root\']/div[1]/div[1]/div[3]/div[1]/div[1]',
   datasetDivClass: '.study-viewer__title',
-  detailedButtonXPath: '.ant-collapse-header',
-  modalDivClass: '.ant-modal-header',
+  detailedButtonXPath: '.ant-collapse-header:nth-child(1)',
+  detailedButtonXPath1: '//body/div[@id=\'root\']/div[1]/div[1]/div[3]/div[1]/div[1]/div[2]/div[2]/div[1]/'
+      + 'div[2]/div[1]/div[1]/div[1]',
+  modalDivClass: '.ant-modal-content',
+  backButton: '.back-link',
 
   learnMoreButtonXPath: '//button[contains(text(), \'Learn More\')]',
   loginRAButtonXPath: '//button[contains(text(), \'Login to Request Access\')]',
   requestAccessButtonXPath: '//button[contains(text(), \'Request Access\')]',
-  downloadButtonXPath: '//button[contains(text(), \'Download\')]',
-  modalDownloadButton: '//a[contains(text(), \'download\')]',
+  // //button[contains(text(), 'Download')]
+  downloadButtonXPath: '.g3-button--primary',
+  modalDownloadButton: '//a[contains(text(),\'download\')]',
   closeButton: '//button[contains(text(), \'Close\')]',
+  // TODO : improve the selector
+  disabledButton: '//*[@id="root"]/div/div/div[3]/div/div/div/div[2]/div/div[1]/div/div[1]/div/div/button',
```

Comment:
how do we know this is a disabled button, and not an enabled button? do they have a different xpath?

Reply:
When the request is in `APPROVED` status, it displays `DAR in PROGRESS` button is displayed. This button is displayed and assertion for `APPROVED` status. But when the request is in `SIGNED` status, the enabled button is `DOWNLOAD` button is displayed. They have different xpaths